frontend/src/components/activity/content/ResponsiveLayout.vue

```diff
@@ -1,119 +1,107 @@
-            :layout-mode="layoutMode"
-            :parent-content-node="contentNode"
-            :disabled="disabled"
-            direction="row"
-          />
-        </div>
-        <v-sheet
-          v-if="!layoutMode && isDefaultVariant && hasAsideTop && hasAsideBottom"
-          tile
-          class="flex-grow-1"
+          {{ $tc('contentNode.responsiveLayout.printAboveMainContent') }}
+        </p>
+        <draggable-content-nodes
+          slot-name="aside-top"
+          :layout-mode="layoutMode"
+          :parent-content-node="contentNode"
+          :disabled="disabled"
+          direction="row"
         />
-        <div
-          v-if="hasAsideBottom || layoutMode"
-          class="d-flex flex-column ec-responsivelayout__slot"
-          :class="{ 'flex-grow-1': !layoutMode }"
+      </div>
+      <v-sheet
+        v-if="!layoutMode && !(hasAsideTop && hasAsideBottom)"
+        tile
+        class="flex-grow-1"
+      />
+      <div
+        v-if="hasAsideBottom || layoutMode"
+        class="d-flex flex-column ec-responsivelayout__slot"
+        :class="{ 'flex-grow-1': !layoutMode }"
+      >
+        <p
+          v-if="layoutMode"
+          class="mt-2 mb-1 blue--text text--darken-3 font-weight-medium"
         >
-          <p v-if="layoutMode" class="text-center">
-            {{ $tc('contentNode.responsiveLayout.printBelowMainContent') }}
-          </p>
-          <draggable-content-nodes
-            slot-name="aside-bottom"
-            :layout-mode="layoutMode"
-            :parent-content-node="contentNode"
-            :disabled="disabled"
-            direction="row"
-          />
-        </div>
-      </LayoutItem>
+          {{ $tc('contentNode.responsiveLayout.printBelowMainContent') }}
+        </p>
+        <draggable-content-nodes
+          slot-name="aside-bottom"
+          :layout-mode="layoutMode"
+          :parent-content-node="contentNode"
+          :disabled="disabled"
+          direction="row"
+        />
+      </div>
     </div>
-  </LayoutCard>
+  </LayoutNodeCard>
 </template>
 
 <script>
 import { contentNodeMixin } from '@/mixins/contentNodeMixin.js'
 import DraggableContentNodes from '@/components/activity/DraggableContentNodes.vue'
-import LayoutItem from '@/components/activity/content/layout/LayoutItem.vue'
 import MenuCardlessContentNode from '@/components/activity/MenuCardlessContentNode.vue'
-import LayoutCard from '@/components/activity/content/layout/LayoutCard.vue'
+import LayoutNodeCard from '@/components/activity/content/layout/LayoutNodeCard.vue'
 import { groupBy } from 'lodash'
 
-const ASIDE_CONTENT_WIDTH = 400
-const MAIN_CONTENT_WIDTH = 700
+const ASIDE_CONTENT_WIDTH = 250
+const MAIN_CONTENT_WIDTH = 600
 const GAP = 1
 
 export default {
   name: 'ResponsiveLayout',
   components: {
     DraggableContentNodes,
     MenuCardlessContentNode,
-    LayoutCard,
-    LayoutItem,
+    LayoutNodeCard,
   },
   mixins: [contentNodeMixin],
+  inject: ['isPaperDisplaySize'],
```

Comment:
Nice use-case of inject. 👍  
We could think about doing the same with `layoutMode`, to avoid the need to pass it along the whole content node tree.

Reply:
Yes good idea. Could you create a ticket for this?

Reply:
https://github.com/ecamp/ecamp3/issues/4340